canister_ids.json

```diff
@@ -1,6 +1,6 @@
 {
   "internet_identity": {
-    "messaging": "rdmx6-jaaaa-aaaaa-aaadq-cai",
+    "identity": "rdmx6-jaaaa-aaaaa-aaadq-cai",
```

Comment:
Another naive question: Why was this called messaging before? Was this just wrong?

Reply:
We used the `messaging` testnet as our testnet for a while, and the configuration remained afterwards.